Applications/Express.md

```diff
@@ -0,0 +1,140 @@
+## Express
+
+The express project is applying to become an incubating
+[Top Level Project of the Node.js Foundation][].
+
+### About Express, User base, Community and Ecosystem
+
+Express is one of the most popular web server frameworks available for Node.js.
+For many Node.js developers, express is their primary interface with Node.js
+on the server.
+
+Express has consistently been among the top packages installed via npm, with
+nearly 4.5 million downloads in just the past month.
+
+It is important to recognize that Express is not the only web framework in the
+Node.js ecosystem and that by incubating Express, neither the Node.js
+Foundation or Node.js core would be "picking a winner" or endorsing Express
+over any other overlapping or similar project.
+
+### Scope 
+
+The express project is composed of a collection of separate modules and 
+documentation currently spread out over multiple GitHub organizations. This top 
+level project would consist initially of the following components:
+
+* strongloop
+  * strongloop/express 
+* jshttp
+  * jshttp/accepts
+  * jshttp/basic-auth
+  * jshttp/compressible
+  * jshttp/content-disposition
+  * jshttp/content-type
+  * jshttp/cookie
+  * jshttp/compressible
+  * jshttp/etag
+  * jshttp/forwarded
+  * jshttp/fresh
+  * jshttp/http-errors
+  * jshttp/media-typer
+  * jshttp/methods
+  * jshttp/mime-db
+  * jshttp/mime-types
+  * jshttp/negotiator
+  * jshttp/on-finished
+  * jshttp/on-headers
+  * jshttp/proxy-addr
+  * jshttp/range-parser
+  * jshttp/statuses
+  * jshttp/type-is
+  * jshttp/vary
+* expressjs
```

Comment:
What's the problem with just letting the Express TC decide what to do with them?